src/parser/jobs/rpr/modules/Procs.tsx

```diff
@@ -67,6 +69,15 @@ export class Procs extends CoreProcs {
 		},
 	]
 
+	protected override jobSpecificOnProcGainedConsiderEvent(event: Events['statusApply']): boolean {
+		if (event.status === this.data.statuses.ENHANCED_GIBBET.id) {
+			if (this.ignoreEnhancedGibbetUntilTimestamp > this.parser.currentDuration) { return false }
```

Comment:
Functionally, this will still work for being consistent, but the more standard convention for this would be to compare to and ignore based on `event.timestamp`